DSCResources/MSFT_SystemLocale/MSFT_SystemLocale.psm1

```diff
@@ -0,0 +1,148 @@
+#region LocalizedData
+$Culture = 'en-us'
+if (Test-Path -Path (Join-Path -Path $PSScriptRoot -ChildPath $PSUICulture))
+{
+    $Culture = $PSUICulture
+}
+Import-LocalizedData `
+    -BindingVariable LocalizedData `
+    -Filename MSFT_SystemLocale.psd1 `
+    -BaseDirectory $PSScriptRoot `
+    -UICulture $Culture
+#endregion
+
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([Hashtable])]
+    param
+    (
+        [parameter(Mandatory = $true)]
+        [ValidateSet('Yes')]
+        [String] $IsSingleInstance,
+
+        [parameter(Mandatory = $true)]
+        [String] $SystemLocale
+    )
+
+    Write-Verbose -Message ( @(
+            "$($MyInvocation.MyCommand): "
+            $($LocalizedData.GettingSystemLocaleMessage)
+        ) -join '' )
+
+    # Get the current System Locale
+    $CurrentSystemLocale = Get-WinSystemLocale `
+        -ErrorAction Stop
+
+    # Generate the return object.
+    $ReturnValue = @{
+        IsSingleInstance = 'Yes'
+        SystemLocale     = $CurrentSystemLocale.Name
+    }
+
+    return $ReturnValue
+} # Get-TargetResource
+
+function Set-TargetResource
+{
+    [CmdletBinding()]
+    param
+    (
+        [parameter(Mandatory = $true)]
+        [ValidateSet('Yes')]
+        [String] $IsSingleInstance,
```

Comment:
This is the standard single instance design pattern Travis came up with :smiley: 
https://blogs.msdn.microsoft.com/powershell/2015/06/22/want-to-write-a-dsc-resource-where-only-a-single-instance-can-be-configured/

https://msdn.microsoft.com/en-us/powershell/dsc/singleinstance